src/ansys/fluent/core/services/field_data.py

```diff
@@ -1076,13 +1242,22 @@ def get_vector_field_data(
         )
         fields = extract_fields(self._service.get_fields(fields_request))
         vector_field_data = next(iter(fields.values()))
-        return {
-            surface_id: (
-                vector_field_data[surface_id][field_name],
-                vector_field_data[surface_id]["vector-scale"][0],
+
+        if surface_name:
+            return VectorFieldData(
+                surface_ids[0],
+                vector_field_data[surface_ids[0]][field_name],
+                vector_field_data[surface_ids[0]]["vector-scale"][0],
```

Comment:
The "vector-scale" string originates in the FieldData gRPC code. Would it be correct to say that we need to mention this string in the proto file, so that the service can be used effectively?

Reply:
Response only contains payload. We can discuss how to extend.